cumulusci.yml

```diff
@@ -409,6 +417,40 @@ flows:
             5:
                 flow: preferred_email_testing
 
+    lqa_org:
+        description: Set up an org as a testing environment for LQA
+        group: "Org Setup"
+        steps:
+            1:
+                flow: dependencies
+            2:
+                flow: deploy_unmanaged
+            3:
+                flow: config_lqa
+
+    config_lqa:
+        description: Configure an org for LQA with sample data
+        group: "Post-Install Configuration"
+        steps:
+            1:
+                task: deploy_post
+            2:
+                task: update_admin_profile
+            3:
+                task: deploy_dev_config
+            4:
+                task: deploy_qa_config
+            5:
+                task: deploy_lqa_config
```

Comment:
We'll want an execute install apex task in here (thank you @spinkelman) to set up the affiliations.

Reply:
does the order of these tasks matter? i noticed in 2gp config we ran execute install apex task right after update_admin_profile